pkg/build/apis/build/v1/defaults_test.go

```diff
@@ -144,7 +144,7 @@ func TestDefaults(t *testing.T) {
 }
 
 func roundTrip(t *testing.T, obj runtime.Object) runtime.Object {
-	data, err := runtime.Encode(codecs.LegacyCodec(LegacySchemeGroupVersion), obj)
```

Comment:
@deads2k we won't test the legacy api roundtrip anymore?

Reply:
> @deads2k we won't test the legacy api roundtrip anymore?

the default test was different.  Before were only testing legacy, now we only test the version in the package.